config/prod/prow/jobs/custom/peribolos.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 The Knative Authors
```

Comment:
would make more sense to me renaming this file to `knative-sandbox.yaml`

Reply:
The jobs here are running for the `knative/community` repo, so I feel `knative-sandbox.yaml` might not be a good name. How about `peribolos-knative-sandbox.yaml`?

Reply:
Doesn't really matter that much

/lgtm